lib/jbuild

```diff
@@ -0,0 +1,31 @@
+(* -*- tuareg -*- *)
+#require "unix"
+
+let flags pkgs =
```

Comment:
This is a very welcome horrible hack :smile: 
It is practically allowing us to port the rest of `xapi` to `jbuilder` while we wait to be able to make the larger set of changes needed to fully update our upstream librarie :tada: